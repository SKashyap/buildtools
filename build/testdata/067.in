# buildifier:noformat
my_list=[
    ["//abc:def",     "foo",          {11111:  2}],
    ["xyz",           "barbaz",       {33:    44}],
]

my_var="somevalue"
