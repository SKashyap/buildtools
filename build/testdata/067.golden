# buildifier:noformat
my_list=[
    ["//abc:def",     "foo",          {11111:  2}],
    ["xyz",           "bar",          {33:    44}],
    ["xy:z",          "baz",          {663:   64}],
]

my_var = "somevalue"
